addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -236,3 +236,37 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
         color: adjust-color-to-background($text-muted, $dropdown-bg, mute-color($color-contrast-light), mute-color($color-contrast-dark)) !important;
     }
 }
+
+// The following code adapts the style of disabled inputs to maintain color
+// consistency and make them more recognizable.
+.form-select {
+    // This is necessary to achieve a consistent "border rendering behaviour"
+    // across form-select and form-control when using semi-transparent
+    // borders and a background color (eg. disabled inputs).
+    // Otherwise, the color rendering of the semi-transparent border may be altered
+    // (the color may be darker depending on the background color used).
+    background-clip: padding-box;
+
+    &:disabled:not([multiple]):not([size]:not([size="1"])) {
+        background-image: str-replace($form-select-indicator, #{$form-select-indicator-color}, str-replace(#{mix($input-color, $input-bg)}, "#", "%23"));
+    }
+}
+
+.form-control {
+    &:disabled, &[readonly] {
+        color: mix($input-color, $input-bg);
```

Comment:
You use this at some places but this is equal to the value of `$form-select-disabled-color`, should not you uses it ?

Reply:
Good catch ! I've updated the code to use this variable when necessary.